Fix error when running pre-built binary

# Goal

Fix error running pre-built binary

Running from source works, but using the released binary causes the error below.

# Additional information

At run time:
```
terminate called after throwing an instance of 'cv::Exception'
  what():  OpenCV(4.5.1-dev) /home/pi/opencv_src/modules/imgproc/src/color.cpp:182: error: (-215:Assertion failed) !_src.empty() in function 'cvtColor'
```

--- a/PosturePerfection_install.sh
+++ b/PosturePerfection_install.sh
@@ -11,7 +11,7 @@ mkdir -p models && cd models
 wget https://github.com/ESE-Peasy/PosturePerfection/raw/main/models/EfficientPoseRT_LITE.tflite
 cd ..
 mkdir -p images && cd images
-wget https://github.com/ESE-Peasy/PosturePerfection/blob/main/images/logo.png  
+wget https://github.com/ESE-Peasy/PosturePerfection/raw/main/images/logo.png  
 cd ..
 
 # Download and install the PosturePerfection build of OpenCV

